ros_ws/src/navigation_stack/navigation_stack_config/config/costmap_common_params.yaml

```diff
@@ -8,9 +8,9 @@ obstacle_layer:
  enabled: true
  obstacle_range: 3.0
  raytrace_range: 3.5
- inflation_radius: 0.2
```

Comment:
Because the inflation radius is missing the global planner creates a path much too close to the wall. This might be problematic with an actual car; driving without a crash should be the highest priority.